buildTree.py

```diff
@@ -0,0 +1,32 @@
+Time complexity: O(n)
```

Comment:
Time should be O(n^2). Getting index is O(n) and you are having that operation for n elements.